django/contrib/redirects/forms.py

```diff
@@ -0,0 +1,42 @@
+from django import forms
+from django.apps import apps
+from django.contrib import admin
+from django.contrib.redirects.models import Redirect
+from django.utils.translation import gettext_lazy as _
+
+
+class RedirectForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if apps.is_installed('django.contrib.sites'):
+            from django.contrib.sites.models import Site
+
+            self.fields['site'] = forms.ModelChoiceField(
+                queryset=Site.objects.all(),
+                to_field_name='domain',
+                empty_label='Any',
+                widget=admin.widgets.AdminRadioSelect(attrs={
```

Comment:
This will only work for up to 3-5 sites. I would recommend using a select input.

Reply:
Select is less convenient when there are not many sites, which I believe is the vast majority of applications. Also, it was the radio select before, so nothing became worse here at least ^^

Reply:
hm... ok. fair enough, maybe it makes sense to make it swappable, but I don't want to overcomplicate things.